src/utils/database.ts

```diff
@@ -6221,6 +6222,28 @@ const createQuickPoll = async (
 
     if (participantsError) throw participantsError
 
+    // Send invitation emails to all participants (excluding the host)
+    const inviterName =
+      ownerAccount.preferences?.name || ellipsizeAddress(owner_address)
+
+    for (const participant of invitees) {
+      if (participant.guest_email) {
+        try {
+          await sendPollInviteEmail(
```

Comment:
This would lead to a too many requests error in production. Implement this with the EmailQueue worker at utils/workers/email.queue.ts and reference notification_helper.ts for its implementation